### PR TITLE
docs: customize sphinx doc theme

### PR DESCRIPTION
**Pull request by sparkbrains**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 * Checks that titles and narrative text are at least 50% alpha characters.
 * Restricts titles to a maximum word length. Adds a `UNSTRUCTURED_TITLE_MAX_WORD_LENGTH`
   environment variable for controlling the max number of words in a title.
-
+* Adding a feature to customizing the color scheme of sphinx documentation.
```
Comment (MthwRobinson) [inline CHANGELOG.md:16]: Don't need a change log entry for this one since the source code didn't change.

**Review by MthwRobinson — CHANGES_REQUESTED** [review]
You can drop the `CHANGELOG` entry since the source code didn't change. Can you also update the title to: `docs: customize sphinx doc theme`?

**Comment by MthwRobinson** [comment]
Are you able to add the same color scheme to the top part with "unstructured" and the search bar?

image

**Comment by sparkbrains** [comment]
> unstructured

Yes I was able to setup the same color for both sidebar and the search bar.

File: CHANGELOG.md (hunk 1 above)
Comment (sparkbrains) [reply CHANGELOG.md]: This has been removed

**Review by MthwRobinson — APPROVED** [review]
LGTM - thanks!

**Pull request merged**